include/boost/url/grammar/recycled.hpp

```diff
@@ -90,8 +90,14 @@ class recycled
     {
         T t;
         U* next = nullptr;
+
+#if !defined(BOOST_URL_DISABLE_THREADS)
```

Comment:
This is unnecessary. Single threaded C++11 platforms have a working <atomic>.

Reply:
`<atomic>`